DFC.Api.JobProfiles.IntegrationTests/Model/DetailsAPIResponse/College.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
```

Comment:
Are we able to make the generic objects better defined to the actual types we are expecting back?

Reply:
Understood. I'll look to change object types on models.

Reply:
Done.